recipe/recipe.yaml

```
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  version: "0.9.11"
+  cov_fail_under: "91"
+
+package:
+  name: python-crfsuite
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/p/python-crfsuite/python_crfsuite-${{ version }}.tar.gz
+  sha256: 6eff965ca70567396d822c9a35ea74b0f7edb27d9471524997bdabe7a6da5f5a
+
+build:
+  number: 3
+  script:
+    file: build-pycrfsuite
+
+requirements:
+  build:
+    - ${{ compiler("c") }}
+    - ${{ compiler("cxx") }}
+    - ${{ stdlib("c") }}
+    - if: build_platform != target_platform
+      then:
+        - python
+        - cross-python_${{ target_platform }}
```

```suggestion
        - cross-python_${{ target_platform }}
        - cython
```

don't you need cython as well here?

---

let's try!

---

(and also, thank you for the speedy and helpful review... you know me, just shaving yaks)